plugins/org.jkiss.dbeaver.ext.greenplum/plugin.xml

```diff
@@ -74,6 +82,62 @@
                     <property name="prepareThreshold" value="0"/>
                 </driver>
 
+                <provider-properties drivers="*">
```

Comment:
Let's create new provoder property inheritProviderProperties. And remove copy-paste in existing PG-based drivers